composer.json

```diff
@@ -1,32 +1,35 @@
 {
-    "name": "andreeasabadis/quiz-app",
-    "authors": [
-        {
-            "name": "Andreea Sabadis",
-            "email": "andreea.sabadis@evozon.com"
-        }
-    ],
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "git@github.com:andreiuta97/mvc-framework-skeleton.git"
-        },
-        {
-            "type": "vcs",
-            "url": "git@github.com:andreiuta97/reallyOrm.git"
-        },
-        {
-            "type": "vcs",
-            "url": "git@github.com:andreiuta97/codeHighlightLib.git"
-        }
-    ],
-    "require": {
-        "andreeasabadis/mvc-framework-skeleton": "dev-master",
-        "really-orm-skeleton": "dev-master",
-        "andreeasabadis/code-highlight-lib": "dev-master",
-      "ext-pdo": "*"
+  "name": "andreeasabadis/quiz-app",
+  "authors": [
+    {
+      "name": "Andreea Sabadis",
+      "email": "andreea.sabadis@evozon.com"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "git@github.com:andreiuta97/mvc-framework-skeleton.git"
+    },
+    {
+      "type": "vcs",
+      "url": "git@github.com:andreiuta97/reallyOrm.git"
     },
-    "autoload": {
-        "psr-4": {"QuizApp\\": "src/"}
+    {
+      "type": "vcs",
+      "url": "git@github.com:andreiuta97/codeHighlightLib.git"
+    }
+  ],
+  "require": {
+    "andreeasabadis/mvc-framework-skeleton": "dev-master",
+    "really-orm-skeleton": "dev-master",
+    "andreeasabadis/code-highlight-lib": "dev-master",
+    "ext-pdo": "*",
+    "php": "7.2.5"
```

Comment:
`php` key should be first in `"require"`. after the php key, you add the extensions, and after everything else, you add the dependencies

Reply:
Done in master (I hope it's not a problem) :)